### PR TITLE
DM-7178: apply formatting to policy yaml dump

### PR DESCRIPTION
**Pull request by n8pease**

### CODE REVIEW

File: python/lsst/daf/persistence/policy.py [hunk 1]
```diff
@@ -437,7 +437,16 @@ def dump(self, output):
         :return:
         """
         data = copy.copy(self.data)
-        yaml.dump(data, output)
+        keys = ['defects', 'needCalibRegistry', 'levels', 'defaultLevel', 'defaultSubLevels',
```
Comment (timj) [inline python/lsst/daf/persistence/policy.py]: Please add a comment before this list to explain that you are specifically handling a subset of the allowed keys to do something special with them and then will dump the remainder in the standard way.

**Comment by n8pease** [comment]
this was merged last week. I don't know why it didn't automatically close the PR.

from master in daf_persistence:

commit 81c78f3a1e81a5c4a4b47189ed31bd94ea64d4e4
Merge: e3e91d9 38be87d
Author: Nate Pease <n8pease@gmail.com>
Date:   Mon Aug 22 17:09:53 2016 -0700

    Merge branch 'tickets/DM-7178'

**Comment by timj** [comment]
Github will not spot a merge outside of github if you do something like the following:

* rebase ticket branch
* checkout master
* merge ticket branch
* push to github

It will only notice if you instead do:

* rebase ticket branch
* push updated branch to github
* checkout master
* merge ticket branch
* push to github

**Comment by timj** [comment]
The relevant merge commit: 81c78f3a1e81a5c4a4b47189ed31bd94ea64d4e4

**Pull request closed without merging**